Gateway/Request/DescriptionDataBuilder.php

```diff
@@ -29,13 +29,7 @@ public function build(array $buildSubject)
         $payment = $paymentDataObject->getPayment();
         /** @var Order $order */
         $order = $payment->getOrder();
-
-        $request['body']['description'] = __(
-            'Order %1 from %2',
-            $order->getIncrementId(),
-            $order->getStore()->getGroup()->getName()
-        );
-
+        $request['body']['description'] = 'Order ' . $order->getIncrementId() . ' from ' . $order->getStore()->getGroup()->getName();
```

Comment:
We will lose the localisation capabilities after removing the phrase and merchants will not be able to add translation. Do we know why this is an issue now especially after migrating to model based library?